src/MainWindow.vala

```diff
@@ -201,6 +201,15 @@ namespace PantheonCalculator {
             var button_tanh = new Button ("tanh", _("Hyperbolic Tangent"));
             var button_pi = new Button ("π", _("Pi"));
             var button_e = new Button ("e", _("Euler's Number"));
+            var button_log = new Button ("log", _("log 2 base 10"));
+            var button_ln = new Button ("ln", _("log base e"));
+            var button_asin = new Button ("sin<sup>-1</sup>", _("arc sine"));
+            button_asin.function = "asin";
+            var button_acos = new Button ("cos<sup>-1</sup>", _("arc cosine"));
+            button_acos.function = "acos";
+            var button_atan = new Button ("tan<sup>-1</sup>", _("arc tangent"));
```

Comment:
The tooltips should be in title case:  "Logarithm Base 10", "Natural Logarithm", "Inverse Sine", "Inverse Cosine", "Inverse Tangent".   ("Arcsine", "Arccosine" and "Arctangent" are also OK)